cellprofiler/library/functions/library_utils.py

```diff
@@ -0,0 +1,296 @@
+from re import S
+import numpy
+import skimage
+import scipy
+import centrosome
+import cellprofiler.library as cp
```

Comment:
We've generally decided against aliasing in most places in the code, to increase readability, though that's something we can certainly readdress